larray_editor/editor.py

```diff
@@ -493,6 +494,9 @@ def ipython_cell_executed(self):
                     if isinstance(cur_output, matplotlib.axes.Subplot) and 'inline' not in matplotlib.get_backend():
                         show_figure(self, cur_output.figure)
 
+                    if isinstance(cur_output, collections.Iterable) and isinstance(np.ravel(cur_output)[0], matplotlib.axes.Subplot):
```

Comment:
you should also check for the inline matplotlib backend, as above and both tests (the one for a single instance above and this one) could be somewhat combined.